pkg/controller/endpointslicemirroring/metrics.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package metrics
+package endpointslicemirroring
```

Comment:
why removing this from its own folder?

Reply:
I don't know, I feel that is more readable in its own folder, all the metrics are prefixed.
I defer to @robscott for this